manifests/0000_90_kube-apiserver-operator_04_servicemonitor-apiserver.yaml

```diff
@@ -135,3 +135,14 @@ spec:
     - record: cluster:apiserver_current_inflight_requests:sum:max_over_time:2m
       expr: |
         max_over_time(sum(apiserver_current_inflight_requests{apiserver=~"openshift-apiserver|kube-apiserver"}) by (apiserver,requestKind)[2m:])
+  - name: pre-release-lifecycle
+    rules:
+      - alert: DeprecratedAPIUsed
+        annotations:
+          message: |
+            Deprecated API that will be removed in the next version is being used: {{"{{$labels.group}}"}}.{{"{{$labels.version}}"}}/{{"{{$labels.resource}}"}}
```

Comment:
We mentioned to also say how to fix this, e.g. "Remove the workload that is using this API, otherwise <insert reason what happens>" anything that is helpful to end users, who receive this page.

Also is it not possible to get a namespace for this to display?

Reply:
Updated to:
> Deprecated API that will be removed in the next version<sup>1</sup> is being used. Removing the workload that is using the `group.version/resource` API might<sup>2</sup> be necessary for a successful upgrade to the next cluster version.
---
<sup>1</sup> by _next version_ here we mean next y-stream version. Is there a more exact, customer-facing terminology to use here to make it clear that a z-stream version would not remove an API? For example, _next minor version_, but the term _minor version_ is not well defined in the documentation.
<sup>2</sup> I say _might_ here because the deprecated API usage might be in the kube control plane itself, which of course would be updated in the next version to not use the deprecated APIs, but I can't easily filter that use in the metric/alert.

Reply:
> Also is it not possible to get a namespace for this to display?
@lilic The namespace is not available.